src/app/settings/add-credit.component.ts

```diff
@@ -47,7 +47,7 @@ export class AddCreditComponent implements OnInit {
 
     constructor(private userService: UserService, private apiService: ApiService,
         private platformUtilsService: PlatformUtilsService) {
-        if (platformUtilsService.isDev()) {
+        if (process.env.ENV !== 'production') {
```

Comment:
Why use `ENV` for this check vs. `NODE_ENV`? Would it perhaps make more sense to use `ENV || NODE_ENV` for all checks?

Reply:
It's been a topic of discussion between me and @joseph-flinn. I've changed it to `env && node_env`. But feel free to change it if we think it should be something else.